lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/VerifiableCredentialService.java

```diff
@@ -142,23 +159,30 @@ private Object[] convertBirthDates(List<BirthDate> birthDates) {
     }
 
     private Object[] calculateEvidence(KBVItem kbvItem) {
-
         Evidence evidence = new Evidence();
         evidence.setTxn(kbvItem.getAuthRefNo());
-
-        EventProbe eventProbe = new EventProbe();
-
-        if (VC_THIRD_PARTY_KBV_CHECK_PASS.equalsIgnoreCase(kbvItem.getStatus())) {
+        if (hasMultipleIncorrectAnswers(kbvItem)) {
+            evidence.setVerificationScore(VC_FAIL_EVIDENCE_SCORE);
+            evidence.setCi(new ContraIndicator[] {ContraIndicator.V03});
+            logVcScore("fail");
+        } else if (VC_THIRD_PARTY_KBV_CHECK_PASS.equalsIgnoreCase(kbvItem.getStatus())) {
             evidence.setVerificationScore(VC_PASS_EVIDENCE_SCORE);
-            eventProbe.addDimensions(Map.of(METRIC_DIMENSION_KBV_VERIFICATION, "pass"));
-            LOGGER.info("kbv pass");
-
+            logVcScore("pass");
         } else {
             evidence.setVerificationScore(VC_FAIL_EVIDENCE_SCORE);
-            eventProbe.addDimensions(Map.of(METRIC_DIMENSION_KBV_VERIFICATION, "fail"));
-            LOGGER.info("kbv fail");
+            logVcScore("fail");
         }
 
         return new Map[] {objectMapper.convertValue(evidence, Map.class)};
     }
+
+    private boolean hasMultipleIncorrectAnswers(KBVItem kbvItem) {
+        return VC_THIRD_PARTY_KBV_CHECK_NOT_AUTHENTICATED.equalsIgnoreCase(kbvItem.getStatus())
+                && kbvItem.getQuestionAnswerResultSummary().getAnsweredIncorrect() > 1;
```

Comment:
It makes sense to leave the `null` check in here that you previously had i.e.
`&& Objects.nonNull(kbvItem.getQuestionAnswerResultSummary())`